_data/scala-compiler-team.yml

```diff
@@ -0,0 +1,97 @@
+# - name: John Doe
+#   description: Knows everything
+#   github: scala
+
+- name: Martin Odersky
+  description: Language Creator
+  github: odersky
+
+- name: Piotr Chabelski
+  description: Product Manager, Specialised Engineer (Scala CLI)
+  github: Gedochao
+
+- name: Tomasz Godzik
+  description: VirtusLab Compiler Team Leader, Specialised Engineer (IDE)
+  github: tgodzik
+
+- name: Oliver Bračevac
+  description: LAMP Compiler Team Leader, Specialised Engineer (Capture Checking)
+  github: bracevac
+
+- name: Dale Wijnand
+  description: Compiler Engineer, Specialised Engineer (Matching)
+  github: dwijnand
+
+- name: Matt Bovel
+  description: Compiler Engineer
+  github: mbovel
+
+- name: Eugene Flesselle
+  description: Compiler Engineer
+  github: EugeneFlesselle
+
+- name: Hamza Remmal
+  description: Infrastructure Maintainer
```

Comment:
```suggestion
  description: Compiler Engineer, Infrastructure Maintainer
```

Reply:
Makes sense! Sorry I didn't add it before!